### ISSUE TITLE
Create new Static Webapp not setting local fs changes

### ISSUE THREAD

**Issue by orta** [microsoft/vscode-azurestaticwebapps#101]
Hey folks! First off, very cool! I'm looking forwards to moving the TypeScript website to static web apps.

I ran through the "Create new Static Webapp" with all the defaults, which created this repo:  https://github.com/orta/devdiv-onboarding/commit/d497d75bb5f17336116ef12bc5d6039d2e2dec50 and also my Azure stuff

![Screen Shot 2020-05-19 at 2 03 46 PM](https://user-images.githubusercontent.com/49038/82361940-9087e700-99d9-11ea-9ca3-aa58f4087c6d.png)

What it didn't do was set up any folders locally, and left me with this screen:

![Screen Shot 2020-05-19 at 2 01 11 PM](https://user-images.githubusercontent.com/49038/82361710-3850e500-99d9-11ea-81bf-f7fa8cefcc7c.png)

It looks fine in the logs:

```sh
1:53:53 PM: Creating resource group "DevDivOnboarding" in location "centralus"...
1:53:55 PM: Successfully created resource group "DevDivOnboarding".
1:53:55 PM: Creating new GitHub repository "devdiv-onboarding"
1:53:57 PM: Created new GitHub repository "devdiv-onboarding"
1:54:02 PM: Created static web app "DevDivOnboarding"
```

---

I guess my expectations were that there would be _something_ after creating. I assume that this did all the GH/Azure bits and then it is on me to clone the repo and do some work, which is fine, but I didn't feel prompted to do so next

**Comment by EricJizbaMSFT** [comment]
@orta can you walk me through your expected steps? You mentioned your "TypeScript website" - does that mean you already tested your app locally before creating a static web app in Azure? And was it a git repo already (just not in GitHub)? I'm trying to figure out how users expect to make the transition from a local app to an app deployed to the cloud.

**Comment by orta** [comment]
That's an aside. 

I was expecting that after running through the template it would clone the repo I had made and open it.

**Comment by nturinski** [comment]
This PR somewhat address this concern https://github.com/microsoft/vscode-azurestaticwebapps/pull/264

It'll tell the user to push to the GitHub repository to trigger another deployment.  There's also a context menu action on the SWA tree item to clone the repo down.  Hopefully this will help guide users on the next steps.

**Comment by nturinski** [comment]
@orta Just wanted to let you know that we released v0.8.0 that streamlines this process a lot more effectively and that Static Web Apps the service also just GA'ed today! 🎉 Hope you still find it very cool.
